pkgs/development/python-modules/compdb/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "compdb";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Sarcasm";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0ssq26hp7590zpygxmbyx5z0mqbcajbk94rwlipymj9qsam9s8wy";
+  };
+
+  checkPhase = ''
```

Comment:
Could this be replaced with just adding `checkInputs = [ pytestCheckHook ];`. It should execute the tests automatically

Reply:
Applied the suggestion!